src/main/java/com/selfxdsd/todos/Puzzle.java

```diff
@@ -411,8 +452,10 @@ public String getTime() {
 
                 @Override
                 public String issueTitle() {
-                    final String[] path = this.getFile().split("/");
-                    final String fileName = path[path.length - 1];
+                    final String fileName = Paths
```

Comment:
@amihaiemil extracted the file name with `Path` just in case `file` might use a different separator.